libphx/src/shader.rs

```diff
@@ -205,20 +216,27 @@ pub unsafe extern "C" fn Shader_Create(
     vs: *const libc::c_char,
     fs: *const libc::c_char,
 ) -> Box<Shader> {
+    shader_create(&vs.convert(), &fs.convert())
+}
+
+pub unsafe fn shader_create(vs: &str, fs: &str) -> Box<Shader> {
```

Comment:
I think this would be nicer if it was a method `Shader::new()` that returns a `Shader`, then in `Shader_Create`, we replace it with:
```rust
 Box::new(Shader::new(&vs.convert(), &fs.convert()))
 ```

Reply:
Next PR